src/models.tsx

```diff
@@ -76,25 +81,20 @@ const Archive = view((get) => {
 
   const content = get(videos)
   if (!content) {
-    return (
-      <>
-        {screen}
-        <ActivityIndicator />
-      </>
```

Comment:
removed spinner because loading is like 50ms on my localhost

Reply:
now that I think about it, maybe we should revert it back? when update is sent over normal network in production, it won't be that instantaneous and so, this might be helpful!